CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Change log for Microsoft365DSC
 
 # 1.23.719.1
-
```

Comment:
Please keep the empty line under the header

Reply:
noted